pkg/interfaces/service.go

```diff
@@ -33,6 +33,13 @@ type DeviceServiceSDK interface {
 	// GetDeviceByName returns the Device by its name if it exists in the cache, or returns an error.
 	GetDeviceByName(name string) (models.Device, error)
 
+	//  DriverConfigs retrieves the driver specific configuration
+	DriverConfigs() map[string]string
+
+	// SetDeviceOpState sets the operating state of device
+	// Custom functionality or macros
```

Comment:
Remove this comment

Reply:
the comment below has been removed

// Custom functionality or macros